WordPress/src/main/java/org/wordpress/android/ui/sitecreation/domains/NewSiteCreationDomainsViewModel.kt

```diff
@@ -146,13 +152,15 @@ class NewSiteCreationDomainsViewModel @Inject constructor(
                 }
             }
         } else {
+            tracker.trackErrorShown(ERROR_CONTEXT, INTERNET_UNAVAILABLE_ERROR)
             updateUiStateToContent(query, ListState.Error(listState, errorMessageResId = R.string.no_network_message))
         }
     }
 
     private fun onDomainsFetched(query: DomainSuggestionsQuery, event: OnSuggestedDomains) {
         // We want to treat `INVALID_QUERY` as if it's an empty result, so we'll ignore it
         if (event.isError && event.error.type != SuggestDomainErrorType.INVALID_QUERY) {
+            tracker.trackErrorShown(ERROR_CONTEXT, event.error.type.toString(), event.error.message.toString())
```

Comment:
Isn't `event.error.message.toString()` a potential NPE? Also isn't it already a `String`?

```suggestion
            tracker.trackErrorShown(ERROR_CONTEXT, event.error.type.toString(), event.error.message)
```